Adding estimated time until actAdding (#107227)
Azure/azure-content: articles/cost-management-billing/manage/ea-portal-administration.md

#### If you're not an enterprise administrator

> If you're not an enterprise administrator, contact an enterprise administrator to request that they add you to an enrollment. The enterprise administrator uses the preceding steps to add you as an enterprise administrator. After you're added to an enrollment, you receive an activation email. It takes time from account registration to activation (about 5~10 minutes).

Reviewer: ```suggestion
If you're not an enterprise administrator, contact an enterprise administrator to request that they add you to an enrollment. The enterprise administrator uses the preceding steps to add you as an enterprise administrator. After you're added to an enrollment, you receive an activation email. After the account is registered, it's activated in about 5 to 10 minutes.
```
Minor edit for grammar
